src/pug/includes/body.pug

```diff
@@ -45,38 +56,57 @@ include ./fork.pug
 				- difficulty = swag.difficulty + ' difficulty';
 				- difficultyTitle = 'Difficulty: ' + swag.difficulty;
 				- expired = swag.tags.includes('expired');
+				- reference = 'https://devswag.io' + swag.reference ? swag.reference : swag.name;
```

Comment:
I can't understand why the applause-button `url` argument are not prefixed by `https://devswag.io`...
Anyway, something like this would be better if we stick with the prefix:
```suggestion
				- reference = 'https://devswag.io/#' + swag.reference ? swag.reference : swag.name;
```